.gitignore

```diff
@@ -1,7 +1,6 @@
 .DS_Store
 app/dist
-builds/*
-task/builds
+**/builds/*
```

Comment:
I'm not sure i this simplification doesn't come with a price in readibility, I think it fits for prettier and linting but for gitignore I'm not sure of adding ** as prefix